README.md

```diff
@@ -127,6 +173,43 @@ On the Camera Preview Page, the same second button will toggle between
 the full resolution camera view and a one-to-one pixel view intended
 to aid in focusing the camera (the camera is focused manually).
 
+# Dencam Pages
+
+The Dencam user interface has 5 pages, which are traversed using the 
+top button. Each page is displaying information designated to itself.
+(i.e. Solar Display Page displays information recieved from Sun Saver)
+
+## Networking Information Page
+
+This page displays Raspberry Pi username, wLan0 IP address, and the name of 
+the wifi signal it is connected to.
```

Comment:
maybe instead of "wifi signal" "wifi SSID" or "SSID of the wifi access point" ?  Maybe in common parlance people don't use SSID and maybe we could just say "wifi network"